app/models/micropost.rb

```diff
@@ -0,0 +1,14 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  mount_uploader :picture, PictureUploader
+  validates :user_id, presence: true
+  validates :content, presence: true, length: { maximum: 140 }
+  validate  :picture_size
+  scope, :published -> { order(created_at: :desc) }
```

Comment:
trước dòng này cách 1 dòng trắng ra
```suggestion
  scope, :published -> {order created_at: :desc}
```